pennylane_lightning/src/PyBind.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Xanadu Quantum Technologies Inc.
```

Comment:
How about calling this file `bindings.cpp`? The filename `PyBind.cpp` would suggest that this file contains sources from `pybind11`.